meta-resin-common/recipes-support/resin-mounts/resin-mounts-flasher/etc-dnsmasq.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Resin dnmasq configuration
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=dnsmasq.service connman.service
```

Comment:
Why is this before connman? I don't expect connman to use /etc/dnsmasq. I would expect connman to depend on dnsmasq in turn.